### DISCUSSION TITLE
RabbitMQ trigger, X pod per Y queue items.

### DISCUSSION THREAD

**Discussion by aradhell** [kedacore/keda#3382] [Q&A / Need Help]
Hello,

  I couldn't find in the docs, is it possible to scale deployments like add 5 per 1000 queue item?

  At the moment I use 2000 as queueLength but adding/removing pod is being overhead with a queue around 2M items ( in/out 5k per sec. ) 

```spec:
  scaleTargetRef:
    name: xx
  maxReplicaCount: 600
  minReplicaCount: 1
  triggers:
    - type: rabbitmq
      metadata:
        protocol: amqp
        queueName: xx
        mode: QueueLength
        value: "2000"
```

Thanks in advance!

**Comment by tomkerkhove** [comment]
So you're looking for a "add 5 instances per 1000 queued messages" feature, is that a good summary?
